### DISCUSSION TITLE
Just a suggestion and not a bug

### DISCUSSION THREAD

**Discussion by johnlongo** [okd-project/okd#606] [Q&A] [answered]
Just a suggestion and not a bug.
When you release a new version is it possible to include the exact links to the coreos (my env is vmware) to build a cluster from scratch. I've noticed a lot of times when I want to start from scratch the most current coreos version doesn't work and I have to go back to previous versions to find the one that will. So if you could include those links (the ones you used to test with) in your release I think it would even save you guys a lot of grief too.

**Comment by vrutkovs** [comment, marked as answer]
>exact links to the coreos

You mean the starting image? In general it should be "use latest stable FCOS", but due to https://github.com/openshift/okd/issues/566 we have to use [33.20210217.3.0](https://github.com/openshift/okd/issues/566)

>the ones you used to test with

IPI installer uses [this file](https://github.com/vrutkovs/installer/blob/fcos-release-4.7/data/data/fcos-amd64.json) to find out the starting image. 

In 4.8 installer would get [a special subcommand](https://github.com/vrutkovs/installer/blob/fcos-master-devel/docs/user/overview.md#coreos-bootimages) to find that out
